README.md

```diff
@@ -143,190 +91,57 @@ with Workflow(
-lint                           Run a `lint` process on Hera and report problems
-models                         Generate all the Argo Workflows models
-services                       Generate the services of Hera
-test                           Run tests for Hera
-workflows-models               Generate the Workflows models portion of Argo Workflows
-workflows-service              Generate the Workflows service option of Hera
-```
-
-Also, see the [contributing guide](./CONTRIBUTING.md)!
-
-# Comparison
+See the [examples](./examples/workflows-examples.md) for a collection of Argo workflow construction and submission via Hera!
 
-There have been other libraries available for structuring and submitting Argo Workflows:
-
-- [Couler](https://github.com/couler-proj/couler), which aimed to provide a unified interface for constructing and
-  managing workflows on different workflow engines. It has now been unmaintained since its last commit in April 2022.
-- [Argo Python DSL](https://github.com/argoproj-labs/argo-python-dsl), which allows you to programmatically define Argo
-  worfklows using Python. It was archived in October 2021.
-
-While the aforementioned libraries provided amazing functionality for Argo workflow construction and submission, they
-required an advanced understanding of Argo concepts. When [Dyno Therapeutics](https://dynotx.com) started using Argo
-Workflows, it was challenging to construct and submit experimental machine learning workflows. Scientists and engineers
-at [Dyno Therapeutics](https://dynotx.com) used a lot of time for workflow definition rather than the implementation of
-the atomic unit of execution - the Python function - that performed, for instance, model training.
-
-Hera presents an intuitive Python interface to the underlying API of Argo, with custom classes making use of context
-managers and callables, empowering users to focus on their own executable payloads rather than workflow setup.
-
-<details><summary>Here's a side by side comparison of Hera, Couler, and Argo Python DSL</summary>
+## Requirements
 
+Hera requires an Argo server to be deployed to a Kubernetes cluster. Currently, Hera assumes that the Argo server sits
+behind an authentication layer that can authenticate workflow submission requests by using the Bearer token on the
+request. To learn how to deploy Argo to your own Kubernetes cluster you can follow
+the [Argo Workflows](https://argoproj.github.io/argo-workflows/quick-start/) guide!
 
-You will see how Hera has focused on reducing the complexity of Argo concepts while also reducing the total lines of
-code required to construct the `diamond` example, which can
-be <a href="https://github.com/argoproj/argo-workflows/blob/2a9bd6c83601990259fd5162edeb425741757484/examples/dag-diamond.yaml">
-found in the upstream Argo repository</a>.
+Another option for workflow submission without the authentication layer is using port forwarding to your Argo server
```

Comment:
@elliotgunton can we include an example of local authentication and provide a link to the example in this piece of documentation? I am thinking something like this

```python
from hera.workflows import Workflow, Container
from hera.shared import global_config
from hera.auth import ArgoCLITokenGenerator

global_config.host = "http://localhost:2746"
global_config.token = ArgoCLITokenGenerator

with Workflow(generate_name="local-test-", entrypoint="c") as w:
    Container(
        name="c",
        image="docker/whalesay",
        command=["cowsay", "hello"],
    )
```

Reply:
There isn't actually an `ArgoCLITokenGenerator` example, nor is it in the authentication.md guide. Will add this example with a quick explanation here but wanted to make that note

Reply:
Yea, totally, I was suggesting we create one!